onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphDeserialization.cpp

```diff
@@ -0,0 +1,554 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Where will the deserialization be used within ORT?

Reply:
To verify the serialized graph gated with `DML_ENABLE_SERIALIZATION_DEBUG` build flag inside `DmlGraphFusionHelper::FusePartitionAndRegisterKernel()`.